test_src/ConnectionPoolTest.cpp

```diff
@@ -71,13 +71,13 @@ CPPUNIT_TEST_SUITE_END();
          voltdb::Client client = m_connectionPool->acquireClient("localhost", "program", "password", NULL, 21212);
          m_connectionPool->closeClientConnection(client);
          voltdb::Client client2 = m_connectionPool->acquireClient("localhost", "program", "password", NULL, 21212);
-         bool c = client != client2;
+         bool c = !(client == client2);
```

Comment:
There is no != operator defined in Client, so I can either create that or use the change above.